js/packages/phoenix-mcp/src/supportTools.ts

```diff
@@ -0,0 +1,287 @@
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import z from "zod";
+import { RUNLLM_CONFIG, MCP_CONSTANTS } from "./constants.js";
```

Comment:
if these are only used in this file there's no reason to have them in constants.